core/utils/backoff_ticker.go

```diff
@@ -28,34 +28,37 @@ func NewBackoffTicker(min, max time.Duration) BackoffTicker {
 	}
 }
 
-// Starts the ticker
-func (t *BackoffTicker) Start() {
+// Start - Starts the ticker
+func (t *BackoffTicker) Start() bool {
 	t.Lock()
 	defer t.Unlock()
 
 	if t.isRunning {
-		return
+		return false
 	}
 
 	// Reset the backoff
 	t.b.Reset()
 	go t.run()
 	t.isRunning = true
+	return true
 }
 
 // Stop stops the ticker. A ticker can be restarted by calling Start on a
 // stopped ticker.
-func (t *BackoffTicker) Stop() {
+// Stop returns true if the ticker was actually stopped
+func (t *BackoffTicker) Stop() bool {
 	t.Lock()
 	defer t.Unlock()
 
 	if !t.isRunning {
-		return
+		return false
```

Comment:
Shouldn't this be true since an already stopped timer is considered stopped.

Reply:
i mean, false relates to to the fact that it was not stopped (so the method did not do what it was supposed to)- i mentioned that in the comment above